x-pack/legacy/plugins/lens/public/editor_frame_plugin/embeddable/embeddable.tsx

```diff
@@ -63,7 +63,7 @@ export class Embeddable extends AbstractEmbeddable<LensEmbeddableInput, LensEmbe
       initialInput,
       {
         defaultTitle: savedVis.title,
-        savedObjectId: savedVis.id!,
+        savedObjectId: savedVis.id,
```

Comment:
:information_source: `savedObjectId` actually allows `undefined` so no need to non-null assert here.